MacElsewhen/Views/PreferencesViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  PreferencesViewController.swift
+//  MacElsewhen
+//
+//  Created by David Stephens on 19/11/2021.
+//
+
+import Cocoa
+
+class PreferencesViewController: NSViewController {
+    @IBOutlet weak var shouldShowWidgetCheckbox: NSButton!
+    @IBOutlet weak var shouldTerminateAfterLastWindowCheckbox: NSButton!
+    
+    @IBOutlet weak var defaultTimeZoneButton: NSButton!
+    
+    @IBOutlet weak var systemLocaleRadioButton: NSButton!
+    @IBOutlet weak var twelveHourRadioButton: NSButton!
+    @IBOutlet weak var twentyFourHourRadioButton: NSButton!
+    
+    @IBOutlet weak var mykeModeSeperatorLabel: NSTextField!
+    @IBOutlet weak var hyphenRadioButton: NSButton!
```

Comment:
What do these three outlets connect to if you're creating the buttons in code rather than in Interface Builder?

Reply:
Oops, good spot. I forgot to remove them!